src/modules/search-result/author.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import styles from './author.css'
+
+import { classNames } from 'utils'
+
+const List = ({ children, className, ...htmlProps }) => (
+  <ol className={classNames.use(styles.list).join(className)} {...htmlProps}>
+    {children}
+  </ol>
+)
+
+const Item = ({ children, href, ...htmlProps }) => {
+  const ElementType = href != null ? 'a' : 'span'
+  return (
+    <li
+      itemProp="author"
+      itemScope=""
+      itemType="https://schema.org/Person"
```

Comment:
Maybe it would be better if we handle this in JSONLD. We already use everywhere else but I am leaving this decision up to you.

Reply:
Can you tell more how JSON-LD can be integrated in this the particular case?

Reply:
For example, I did integration here https://github.com/oacore/search/blob/master/pages/data-providers/index.jsx#L87. I don't know I find it easier to use. And I somewhere read that's it's recommended way for adding structured data to the website

Reply:
I see this is quite nice. But I still don't understand how or where you can embed this if you have multiple instances of the same thing on the page or you propose to don't do it?

Reply:
That's something I don't know. 🤷‍♂️